packages/strapi-admin/admin/src/containers/AppLoader/tests/index.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { AppLoader } from '../index';
+
+describe('<AppLoader />', () => {
```

Comment:
More tests need to be added to this component. At least one that testes the shouldLoad instance